test/external/CMakeLists.txt

```diff
@@ -2,13 +2,6 @@
 # Add tests based on external repositories
 #
 
-# FindMPI broke backward compatibility with v3.19
-if(${CMAKE_VERSION} VERSION_LESS "3.10")
-  get_filename_component(MPIEXEC_NAME ${MPIEXEC} NAME)
-else()
-  get_filename_component(MPIEXEC_NAME ${MPIEXEC_EXECUTABLE} NAME)
-endif()
-
 # To work around CI failures that seem related to #894 we should run `mpiexec` not
 # `/path/to/mpiexec`
 set(MPIEXEC_NAME ${MPIEXEC_NAME})
```

Comment:
I'm not sure why I didn't ask about this on the original PR (https://github.com/neuronsimulator/nrn/pull/1053), but isn't this just a NOOP? @pramodk?

Reply:
oh right.. i was not even looking at this just now - but on Thursday when i was staring at this file I thought also that this line makes little sense